project/Pack.scala

```diff
@@ -24,9 +24,9 @@ import xerial.sbt.Pack._
 object Pack extends sbt.Build {
   val daemonClassPath = Seq(
     "${PROG_HOME}/conf",
-    "${PROG_HOME}/lib/daemon/*",
+    "${PROG_HOME}/lib/daemon/*"
     // This is for DFSJarStore
-    "${PROG_HOME}/lib/yarn/*"
+    // "${PROG_HOME}/lib/yarn/*"
```

Comment:
Had to comment out to avoid runtime issues. When I try to submit an example job to Gearpump I get "java.lang.NoSuchMethodError: com.google.common.util.concurrent.Futures.withFallback(". I believe that happens because Gearpump pulls com.google.guava:guava version 11.0.2 from Hadoop dependencies, but Cassandra Java driver I am using needs version 16.0.1. Need to figure out a solution to this.